docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -466,7 +463,7 @@ public class HttpSecurityConfiguration {
 === Shared permission checks
 
 One important rule for unshared permission checks is that only one path match is applied, the most specific one.
-Naturally you can specify as many permissions with the same winning path as you want and they will all be applied.
+When a path wins (matches as the most specific), you can specify multiple permissions for that path and they are all applied.
```

Comment:
Once you say "when a path wins" and have "matches as the most specific" in the brackets and next time you "matches and takes precedence"  and have "wins" in the brackets. I think it is abundant, can you just stick to the "most specific" (matches or whatever you prefer)?